addons/account_invoice_tax/views/account_move_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+
+    <record id="view_move_form" model="ir.ui.view">
+        <field name="name">account.move.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="arch" type="xml">
+            <field name="amount_by_group" position="before">
+                <div colspan="2">
+                    <button name="%(account_invoice_tax.action_view_account_invoice_tax)d" type="action" string="Add" title="Add Tax" class="oe_right oe_link oe_edit_only" attrs="{'invisible': [('type', 'not in', ('in_invoice', 'in_refund', 'out_receipt', 'in_receipt'))]}" context="{'type': type, 'type_operation': 'add'}"/>
```

Comment:
please also make it visible only on draft state

Reply:
no this view is seams that add button would be before remove, but on runbot I see first remove (check this image) It would be nicer if first we show add (on left) and then remove (on right). Perhaps it's related to the class oe_right?

![image](https://user-images.githubusercontent.com/3016656/85870205-2239ff80-b7a3-11ea-878d-d2fe76f9741f.png)

Reply:
@jjscarafia  I have made the suggested changes:

1. You were right, this happens because the oe_right, I have changed and move it the oe_right to the main div of the buttons an now is properly order: first add and then remove button
2. I also make the change of only show it in draft state, as you can see I took advantage of the div and move the attrs there to make the code clearer,
3. I change the attrs to only show in in_invoice, in_refund, in_receipt, before we were also showing the button in the out_receipt.

Best Regards